apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -173,30 +173,35 @@ export class GetStartedPage extends React.Component<any, any> {
 
         <div className={styles.instructionsSection}>
           <h2 id="react">Get started with Fabric React</h2>
-          <p>
-            Use NPM to get Fabric components and core styling. All you need is{' '}
-            <a className={styles.getStartedLink} href="https://nodejs.org/en/">
-              node.js
-            </a>{' '}
-            and{' '}
-            <a className={styles.getStartedLink} href="http://gulpjs.com/">
-              gulp
-            </a>
-            .
-          </p>
+          <p>All you need to get started with Fabric is Node.js and npm.</p>
 
           <ol className={styles.steps}>
             <li>
-              <p>To install the Fabric React NPM package, from the root of your project, run:</p>
+              <p>
+                Install Node.js (at least 8.x) from the
+                <a href="https://nodejs.org">Node.js website</a>.
+              </p>
+            </li>
+            <li>
+              <p>
+                Use npm to create a React app that uses TypeScript. Consider using Facebook's{' '}
+                <a href="https://github.com/facebook/create-react-app">Create React App</a> to get started quickly:
+              </p>
+              <CodeBlock language="bash" isLightTheme={true}>
+                {`npx create-react-app my-app --typescript`}
```

Comment:
unless it auto moves you into the project, don't forget a second line with `cd my-app`, or tacked onto the first line `&& cd my-app`

Reply:
Ah yes, thanks for the reminder! Will do.